lib/helper/REST.js

```diff
@@ -219,5 +219,175 @@ class REST extends Helper {
 
     return this._executeRequest(request);
   }
+
+
+  /**
+ * Checks if the response code is equal to what was provided
+ *
+ * ```js
+ * I.seeResponseCodeIs(200);
+ * ```
+ *
+ * @param {string|number} statusCode
+ */
+  seeResponseCodeIs(statusCode) {
+    expect(this.response.status).to.equal(statusCode, `Status code ${this.response.status} of response is not equal to ${statusCode}`);
+  }
+
+  /**
+   * Checks if the response code is not equal to what was provided
+   *
+   * ```js
+   * I.dontSeeResponseCodeIs(200);
+   * ```
+   *
+   * @param {string|number} statusCode
+   */
+  dontSeeResponseCodeIs(statusCode) {
+    expect(this.response.status).to.not.equal(statusCode, `Status code ${this.response.status} of response is equal to ${statusCode}`);
+  }
+
+  /**
+   * Checks if the response code is an information response code by checking if its value is between 100-199
+   *
+   * ```js
+   * I.seeResponseWasInformation();
+   * ```
+   */
+  seeResponseWasInformation() {
+    expect(this.response.status).to.be.above(99);
+    expect(this.response.status).to.be.below(200);
+  }
+
+  /**
+   * Checks if the response code is a sucessful response code by checking if its value is between 200-299
+   *
+   * ```js
+   * I.seeResponseWasSucessful();
+   * ```
+   */
+  seeResponseWasSucessful() {
+    expect(this.response.status).to.be.above(199);
+    expect(this.response.status).to.be.below(300);
+  }
+
+  /**
+   * Checks if the response code is a redirect response code by checking if its value is between 300-399
+   *
+   * ```js
+   * I.seeResponseWasRedirected();
+   * ```
+   */
+  seeResponseWasRedirected() {
+    expect(this.response.status).to.be.above(299);
+    expect(this.response.status).to.be.below(400);
+  }
+
+  /**
+   * Checks if the response code is an error response code by checking if its value is between 400-499
+   *
+   * ```js
+   * I.seeResponseWasClientError();
+   * ```
+   */
+  seeResponseWasClientError() {
+    expect(this.response.status).to.be.above(399);
+    expect(this.response.status).to.be.below(500);
+  }
+
+  /**
+   * Checks if the response code is a server error response code by checking if its value is between 500-599
+   *
+   * ```js
+   * I.seeResponseWasServerError();
+   * ```
+   */
+  seeResponseWasServerError() {
+    expect(this.response.status).to.be.above(499);
+    expect(this.response.status).to.be.below(600);
```

Comment:
here, as well :)

Reply:
same reply as above, if you are wanting a specific range asserting on an exact header is more likely and that method already exists, the use case for asserting on a specific range of well defined and well known status codes seems unlikely. Consumers will likely either want to assert on an exact status code or generically assert they got a good/bad response.